### PR TITLE
Implement createRefuteTransaction using the nitro-transaction library

### PR DESCRIPTION
**Pull request by cf19drofxots**

### CODE REVIEW

File: packages/wallet/package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "lodash": "^4.17.10",
     "magmo-wallet-client": "0.0.3",
     "nanoid": "^1.0.3",
-    "nitro-protocol": "https://github.com/statechannels/nitro-protocol.git#ag-include-contracts",
+    "nitro-protocol": "https://github.com/statechannels/nitro-protocol.git#expose_createRefuteTransaction",
```
Comment (cf19drofxots) [inline packages/wallet/package.json]: Note that I had to update the branch being referenced here as the `createRefuteTransaction` wasn't being exposed through the `transactions` export.

Comment (lalexgap) [reply packages/wallet/package.json]: Hopefully we can get this switched to the mono repo soon

**Comment by lalexgap** [comment]
It looks like `refute` is being eliminated (see https://github.com/statechannels/nitro-protocol/pull/102).

**Comment by cf19drofxots** [comment]
> t looks like refute is being eliminated (see statechannels/nitro-protocol#102).

Yup, saw that. Closing this.

**Pull request closed without merging**